modules/acquia_cms_starter/modules/acquia_cms_demo_content/composer.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "drupal/acquia_cms_demo_content",
+    "type": "drupal-module",
+    "description": "Contains the example content for demonstration.",
+    "require": {
+        "drupal/config_rewrite": "^1.4"
+    },
+    "config": {
+        "allow-plugins": {
+            "composer/installers": true,
+            "cweagans/composer-patches": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "drupal/core-composer-scaffold": true,
+            "ergebnis/composer-normalize": true,
+            "oomphinc/composer-installers-extender": true,
+            "phpro/grumphp-shim": true,
+            "webdriver-binary/binary-chromedriver": true
+        }
+    }
+}
```

Comment:
@vishalkhode1 - I don't think there's any use of submodule composer.json all dependency should be added in starter composer.json itself.